protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ToConnectTestDataGenerator.java

```diff
@@ -22,230 +26,272 @@
+                .put("boolOptional",
+                        new SchemaBuilder(Schema.Type.BOOLEAN).parameter(PROTOBUF_TAG, "27").optional().build())
+                .put("boolWithDefault", new SchemaBuilder(Schema.Type.BOOLEAN).parameter(PROTOBUF_TAG, "28").build())
+                .put("bytes", new SchemaBuilder(Schema.Type.BYTES).parameter(PROTOBUF_TAG, "29").build())
+                .put("bytesWithParam", new SchemaBuilder(Schema.Type.BYTES).parameter(PROTOBUF_TAG, "18924").build())
+                .put("bytesOptional", new SchemaBuilder(Schema.Type.BYTES).parameter(PROTOBUF_TAG, "30").optional().build())
+                .put("bytesWithDefault",
+                        new SchemaBuilder(Schema.Type.BYTES).parameter(PROTOBUF_TAG, "31").build())
+                .put("str", new SchemaBuilder(Schema.Type.STRING).parameter(PROTOBUF_TAG, "32").build())
+                .put("strWithParam", new SchemaBuilder(Schema.Type.STRING).parameter(PROTOBUF_TAG, "13912").build())
+                .put("strOptional", new SchemaBuilder(Schema.Type.STRING).parameter(PROTOBUF_TAG, "33").optional().build())
+                .put("strWithDefault", new SchemaBuilder(Schema.Type.STRING).parameter(PROTOBUF_TAG, "34").build())
+                .build();
+    }
+
+    public static List<Message> getEnumProtobufMessages() {
+        return Arrays.asList(
+                EnumTypeSyntax3.EnumTest.newBuilder().setCorpus(EnumTypeSyntax3.EnumTest.Corpus.UNIVERSAL).build(),
+                EnumTypeSyntax2.EnumTest.newBuilder().setCorpus(EnumTypeSyntax2.EnumTest.Corpus.UNIVERSAL).build()
+        );
+    }
+
+    public static Schema getEnumSchema(String packageName) {
+        return createConnectSchema(
+                "EnumTest",
+                getEnumTypes(),
+                ImmutableMap.of(
+                        "protobuf.package", packageName
+                )
+        );
+    }
+
+    public static Struct getEnumTypesData(String packageName) {
+        final Struct connectData = new Struct(getEnumSchema(packageName));
+
+        connectData
+                .put("corpus", "UNIVERSAL");
+        return connectData;
+    }
+
+    private static Map<String, Schema> getEnumTypes() {
+        return ImmutableMap.<String, Schema>builder()
+                .put("corpus", new SchemaBuilder(Schema.Type.STRING).parameter("protobuf.type", "enum")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.UNIVERSAL", "0")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.WEB", "1")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.IMAGES", "2")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.LOCAL", "3")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.NEWS", "4")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.PRODUCTS", "5")
+                        .parameter("PROTOBUF_SCHEMA_ENUM.VIDEO", "6")
+                        .parameter("ENUM_NAME", "corpus")
```

Comment:
The naming convention is not consistent. Please make them consistent. Good to prefix "PROTOBUF_SCHEMA_ENUM" to be clear.